modules/apps/commerce/commerce-order-web/src/main/resources/META-INF/resources/commerce_order/general.jsp

```diff
@@ -346,8 +350,9 @@ CommerceOrder commerceOrder = commerceOrderEditDisplayContext.getCommerceOrder()
 
 				<div class="col-xl-3">
 					<commerce-ui:info-box
-						actionLabel='<%= LanguageUtil.get(request, Validator.isNull(printedNote) ? "add" : "edit") %>'
+						actionLabel='<%= commerceOrderEditDisplayContext.hasManageCommerceOrdersPermission() ? LanguageUtil.get(request, Validator.isNull(printedNote) ? "add" : "edit") : null %>'
```

Comment:
Is the update check for this also done using MANAGE_COMMERCE_ORDERS in the MVCActionCommand?

Reply:
Yes, this one uses the same permission